src/custom_nodes/east_ocr/README.md

```diff
@@ -45,3 +45,5 @@ It will compile the library inside a docker container and save the results in `l
 | overlap_threshold | a ratio in a range of 0-1 for non-max suppression algorithm. Defines the overlapping ratio to reject detection as duplicated  | 0.3 |
 | debug  | Defines if debug messages should be displayed | false |
 | max_output_batch  | Prevents too big batches with incorrect confidence level. It can avoid exceeding RAM resources | 100 |
+| x_corner_adjustment | Horizontal size expansion level for text images to compensate cut letter | 0 | 
+| y_corner_adjustment | Vertical size expansion level for text images to compensate cut letter | 0 | 
```

Comment:
I'm not sure about that `to compensate cut letter` part. I understand that the model cuts part of the picture and therefore does not recognize the whole text right? I would rephrase that to perhaps give a little bit more context. 

Also what are the units of the adjustment? Pixels? Percentage?

Reply:
For x percentage of box width, for y percentage of box height.

Reply:
Also there is mistake here, default values are:
- x_corner_adjustment: 0.12
- y_corner_adjustment: 0.3

Reply:
Should we change it in source to 0.0 and set defaults for demo in example config.json? @dtrawins